pkg/k8s/watchers/namespace.go

```diff
@@ -79,22 +69,13 @@ func (k *K8sWatcher) namespacesInit(k8sClient kubernetes.Interface, serNamespace
 	)
 
 	k.namespaceStore = namespaceStore
-	k.blockWaitGroupToSyncResources(wait.NeverStop, swgNamespaces, namespaceController, k8sAPIGroupNamespaceV1Core)
+	k.blockWaitGroupToSyncResources(wait.NeverStop, nil, namespaceController, k8sAPIGroupNamespaceV1Core)
 	k.k8sAPIGroups.addAPI(k8sAPIGroupNamespaceV1Core)
 	asyncControllers.Done()
 	namespaceController.Run(wait.NeverStop)
 }
 
 func (k *K8sWatcher) updateK8sV1Namespace(oldNS, newNS *types.Namespace) error {
-	if oldNS == nil || newNS == nil {
-		return nil
-	}
-
-	// We only care about label updates
-	if comparator.MapStringEquals(oldNS.GetLabels(), newNS.GetLabels()) {
```

Comment:
Why is this removed?

Reply:
This is already being compared in the `k8s.EqualV1Namespace(oldNS, newNS)`. I've added a comment in the commit.